official/vision/beta/ops/augment.py

```diff
@@ -33,280 +33,279 @@
 
 from tensorflow.python.keras.layers.preprocessing import image_preprocessing as image_ops
```

Comment:
For one_hot, can you just add a condition here (https://github.com/tensorflow/models/blob/master/official/vision/beta/tasks/image_classification.py#L230) to not convert to one_hot if cutmix is on?